src/Literature/LiteratureHelpers.tsx

```diff
@@ -2,6 +2,7 @@ import { Literatures } from "./Literatures";
 import { dialogBoxCreate } from "../ui/React/DialogBox";
 import { LiteratureName } from "@enums";
 import { Player } from "@player";
+import React from "react";
```

Comment:
I don't think this is actually needed? (You're not referencing any react types.)

Reply:
I'm pretty sure they need to add it to fix this error:
```
'React' must be in scope when using JSX eslintreact/react-in-jsx-scope
'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.ts(2686)
```

Reply:
Ah, I didn't realize it was needed just to use JSX syntax.